docker/bedtools2/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:15.04
+MAINTAINER Andrey V Kartashov "porter@porter.st"
+COPY bedtools /usr/local/bin/bedtools
```

Comment:
Can the version of bedtools in Ubuntu be used instead? http://packages.ubuntu.com/vivid/bedtools